src/pages/atomicdex/api/v20-dev/non_fungible_tokens/get_nft_list/index.mdx

```diff
@@ -0,0 +1,323 @@
+
+Returns a list of the NFTs owned by the user, shown in descending order of the `block_number` value (the block height when the amount or owner changed). If the request is for NFTs on more than one chain, this means that the order may not be chronological. In the case of ERC1155 tokens, the `block_number` will update when additional NFTs are received or when all NFTs are withdrawn, but will generally remain the same if only some NFTs are withdrawn.
+
+<Note>
+  Before using this method, you must first call the [update\_nft](/atomicdex/api/v20-dev/non_fungible_tokens/update_nft/)
+  method to populate/refresh the local database.
+</Note>
+
+### Request Parameters
+
+| Parameter           | Type    | Description                                                                                                                                                             |
+| ------------------- | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| chains              | array   | List of networks to scan for NFTs. Options are: `POLYGON`, `FANTOM`, `ETH`, `BSC`, or `AVALANCHE`.                                                                      |
+| max                 | boolean | Optional, defaults to `false`. If `true`, response will return all NFTs without pagination, and will ignore the `limit` and `page_number` values.                       |
+| limit               | integer | Optional, defaults to `10`. The number of NFTs displayed per page in response.                                                                                          |
+| page\_number        | integer | Optional, defaults to `1`. The page offset for items in response.                                                                                                       |
+| protect\_from\_spam | boolean | Optional, defaults to `false`. If `true`, any potential spam link found in collection name, token name, symbol will be replaced with `URL redacted for user protection` |
+| filters             | object  | Optional. A standard [TokenFilter](/atomicdex/api/v20/#token-filter) object.                                                                                            |
+
+### Response Parameters
+
+| Parameter | Type            | Description                                                                            |
+| --------- | --------------- | -------------------------------------------------------------------------------------- |
+| nfts      | list of objects | A list of standard [NftInfo](/atomicdex/api/common_structures/nfts/#nft-info) objects. |
+| total     | string          | The total number of NFTs in your wallet matching the request filters.                  |
+| skipped   | string          | The number of NFTs in your wallet excluded by the request filters.                     |
```

Comment:
skipped and total are integers

Reply:
fixed

Reply:
![image](https://github.com/KomodoPlatform/komodo-docs-mdx/assets/52405288/0f218223-6b1d-4daa-9e99-d3b5de5a3e6b)
I see that there are two files where we have strings. Are these files used in docs?